vendor/github.com/armPelionEdge/devicedb/Dockerfile.test_local

```diff
@@ -0,0 +1,5 @@
+FROM golang
+ADD . /go/src/github.com/armPelionEdge/devicedb
+RUN cd src/github.com/armPelionEdge/devicedb && go install
+ENTRYPOINT [ "/bin/sh","/go/src/github.com/armPelionEdge/devicedb/start-devicedb-nocluster.sh"]
```

Comment:
If this shell file has `#!/bin/sh` as the first line (good practice for all scripts to have that initial line), the first param for ENTRYPOINT should not be necessary.